src/settings/global_user.rs

```diff
@@ -1,12 +1,21 @@
 use std::env;
+use std::fs;
 use std::path::{Path, PathBuf};
 
 use cloudflare::framework::auth::Credentials;
-use log::info;
+use config;
 use serde::{Deserialize, Serialize};
 
+use crate::settings::{Environment, QueryEnvironment};
 use crate::terminal::emoji;
-use config::{Config, Environment, File};
+
+const DEFAULT_CONFIG_FILE_NAME: &str = "default.toml";
```

Comment:
here's our const! but it's also possible that the "get_config_dir()" function + this should all just be one big const. idk.

Reply:
update: sort of made it so